packages/shared/lib/wallet.ts

```diff
@@ -480,24 +480,13 @@ export async function asyncSyncAccountOffline(account: WalletAccount): Promise<v
     })
 }
 
-export const asyncGetNodeInfo = (accountId: string, url?: string, auth?: IAuth): Promise<INodeInfo> => {
+export const asyncGetNodeInfo = (url?: string, auth?: IAuth): Promise<IStardustNodeInfo> => {
```

Comment:
This should be type `INodeInfoResponse` from `node-info-response.interface.ts` in `@core/network`